voxblox/include/voxblox/integrator/esdf_integrator.h

```diff
@@ -35,7 +36,7 @@ class EsdfIntegrator {
     FloatingPoint default_distance_m = 2.0;
     // For cheaper but less accurate map updates: the minimum difference in
     // a voxel distance, before the change is propagated.
-    FloatingPoint min_diff_m = 0.001;
+    FloatingPoint min_diff_m = 0.0; //0.001;
```

Comment:
Was this causing issues? from memory even a tiny value here can speed things up a lot

Reply:
Oh good call, turned it off while debugging.

Reply:
Quickly double-checked, error completely unaffected but computations 1.2x as fast or so. Definitely putting it back in.